tools/mtrace/mtrace-reader.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Copyright (c) 2022, Intel Corporation. All rights reserved.
+
+#pylint:disable=mixed-indentation
+
+# Tool to stream data from Linux SOF driver "mtrace" debugfs
+# interface to standard output. Plain "cat" is not sufficient
+# as each read() syscall returns log data with a 32bit binary
+# header, containing the payload length.
```

Comment:
If we keep the current mtrace debugfs, we'll need a simple tool like this to stream data out. 

Note the log decoder can be run on the stdout of this tool.

Reply:
For the record, the current debugfs implementation is:
- user space gives a buffer of size X
- kernel reads new data chunk from the slot up to (X - 4) bytes ( or capped at 1020 bytes due to slot size wrapping)
- kernel writes the length of the chunk as u32 to the start of the user buffer
- kernel writes the log data from offset 4 to user buffer
- kernel tell user space that it has exhausted the given buffer (tells that X bytes have been written)
- user space reads first u32 to know the log chunk size in buffer
- user space gives a new buffer since it was told that the previous has been filled up.
- and so on.

This is not `cat to stdout` friendly for sure, but easier to tool it and simpler to implement in kernel.

Reply:
Ack, let's keep this interface.